src/interfaces/packagingInterfacesAndType.ts

```diff
@@ -501,3 +517,25 @@ export const Package1VersionEvents = {
 };
 
 export type PackageXml = Pick<Package, 'types' | 'version'>;
+
+export type PackagePushRequestReportResult = {
```

Comment:
Added report code from @tshopshireSalesforce to resolve conflict

Reply:
Original report PR from Tyler: https://github.com/forcedotcom/packaging/pull/695/files